tests/test_formatters.py

```diff
@@ -988,3 +990,17 @@ def test_add_language_divs_fr_does_not_replace(lang: str):
 )
 def test_remove_language_divs(input: str, output: str):
     assert remove_language_divs(input) == output
+
+
+@pytest.mark.parametrize(
+    "input,output_contains",
+    (
+        ("abc 123", ["abc 123"]),
+        ("Hi,\n[[ga-seal]]\nBye", ["Hi,", "<img", "Bye"]),
+        ("Hi,\n[[ga-seal]]\nBye[[ga-seal]]", ["Hi,", "<img", "Bye"]),
```

Comment:
Is it expected this only replaces once? Seems a little surprising. May want to make sure this limitation is clearly communicated.

Reply:
It replaces all instances - this test is just checking that each string in the list is present in the output. It doesn't check how many times it appears.

Reply:
We just pushed an extra test assertion to make sure it replaces everything as well.

Reply:
Awesome, thanks. My concern was that it wasn't actually getting replaced, the code change and extra assert take care of that.